extensions/functions_rounding.yaml

```diff
@@ -27,3 +27,68 @@ scalar_functions:
           - value: fp64
             name: "x"
         return: fp64
+  -
+    name: "round"
+    description: >
+      Rounding the value `x` to s decimal places.
+
+    impls:
+      - args:
+          - name: rounding
+            options: [ TIE_TO_EVEN, TIE_AWAY_FROM_ZERO, TRUNCATE, CEILING, FLOOR, AWAY_FROM_ZERO,
+              TIE_DOWN, TIE_UP, TIE_TOWARDS_ZERO, TIE_TOWARDS_ZERO, TIE_TO_ODD ]
```

Comment:
TIE_TOWARDS_ZERO is listed twice on this line.

Reply:
Thanks for noticing this, it is a typo.